packages/create-gatsby/package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "create-gatsby",
+  "version": "0.0.0-2",
+  "main": "lib/index.js",
+  "bin": "lib/cli.js",
+  "license": "MIT",
+  "scripts": {
```

Comment:
Could you add a test script here or some instructions in the README on what should run? Not positive how to run this because I seem to get errors trying to run `jest`/`jest --watch` at root 🤔

Reply:
Ah, sorry. Tests in monorepo packages all need to run from the repo root. You can make it run just these tests by running `yarn jest create-gatsby` in the root of the monorepo.